### PR TITLE
docs: making correcting about the test explorer location

### PR DESCRIPTION
**Pull request by Darsh0307**
Closes https://github.com/MicrosoftDocs/visualstudio-docs/issues/9967

### CODE REVIEW

**Comment by Darsh0307** [comment]
Screenshot 2024-01-19 at 2 21 53 AM

**Comment by Darsh0307** [comment]
Screenshot 2024-01-19 at 2 21 53 AM

**Comment by Court72** [comment]
@Mikejo5000

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/test/run-unit-tests-with-test-explorer.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Visual Studio includes the Microsoft unit testing frameworks for both managed an
 
 ## Run tests in Test Explorer
 
-When you build the test project, the tests appear in Test Explorer. If Test Explorer is not visible, choose **Test** on the Visual Studio menu, choose **Windows**, and then choose **Test Explorer** (or press **Ctrl** + **E**, **T**).
+When you build the test project, the tests appear in Test Explorer. If Test Explorer is not visible, choose **Test** on the Visual Studio menu, choose **Test**, and then choose **Test Explorer** (or press **Ctrl** + **E**, **T**).
```
Comment (Mikejo5000) [inline docs/test/run-unit-tests-with-test-explorer.md:28]:
```suggestion
When you build the test project, the tests appear in Test Explorer. If Test Explorer is not visible, choose **Test** on the Visual Studio menu, and then choose **Test Explorer** (or press **Ctrl** + **E**, **T**).
```

**Review by Mikejo5000 — APPROVED** [review]
Thanks for the PR!

**Comment by Mikejo5000** [comment]
#sign-off

**Pull request merged**